lib/rubocop/sketchup/cop/requirements/initialise_entity.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module SketchupRequirements
+      # SketchUp Entity objects (`Face`, `Edge`, `Group` etc) should not be
+      # initialized using the `new` constructor. These objects are managed by
+      # SketchUp. Instead use the `Sketchup::Entities#add_*` methods to create
+      # them.
+      class InitialiseEntity < SketchUp::Cop
+
+        ENTITY_CLASSES = %i[
```

Comment:
Known direct subclasses:

```
AttributeDictionaries, AttributeDictionary, Axes, Behavior, Curve, DefinitionList, Drawingelement, EdgeUse, Layer, LayerFolder, Layers, LineStyle, LineStyles, Loop, Material, Materials, Page, Pages, RenderingOptions, ShadowInfo, Style, Styles, Texture, Vertex
```

And then you have the subclasses of `Drawinglement`:

```
ComponentDefinition, ComponentInstance, ConstructionLine, ConstructionPoint, Dimension, Edge, Face, Group, Image, SectionPlane, Text
```